lib/texts.js

```diff
@@ -124,4 +124,29 @@ ${leaves}`,
 	'err.msg.appRunning': () => 'App is still running',
 	'err.msg.conditionNotSatisfied': count => `Maximum amount of key presses ${count} reached. Condition was not satisfied`,
 	'err.msg.appNotRunning': () => 'Application is not running',
+
+	// interactive test progress status
+	'interactiveProgress.status.openingApp': () => 'Trying to open app...',
+	'interactiveProgress.status.closingApp': () => 'Trying to open app...',
+	'interactiveProgress.status.bootingDevice': () => 'Running the boot sequence defined for the device...',
+	'interactiveProgress.status.needManual': () => 'Paused. Manual action required, check below.',
+	'interactiveProgress.status.recoveringID': () => 'Trying to recover Suitest device ID...',
+	'interactiveProgress.status.waitingForConnectionFromBootstrap': () => 'Waiting for connection from the Suitest app on device...',
+	'interactiveProgress.status.waitingForConnectionFromIL': () => 'Waiting for connection from the instrumentation library...',
+	'interactiveProgress.status.unistallingApp': () => 'Uninstalling app...',
+	'interactiveProgress.status.uploadingAndInstallingApp': () => 'Uploading and installing app...',
+	// interactive test progress code
+	'interactiveProgress.code.blasterError': () => 'Cannot continue: IR blaster missing or incorrectly attached.',
+	'interactiveProgress.code.bootstrappedPlatformError': () => 'Cannot continue: Suitest bootstrap app is not running.',
+	'interactiveProgress.code.testQueued': () => 'Execution will start as soon as other tests queued on this device will finish execution.',
+	'interactiveProgress.code.candyBoxOffline': () => 'Cannot continue: CandyBox controlling this device is offline.',
+	'interactiveProgress.code.suitestDriveOffline': () => 'Cannot continue: SuitestDrive controlling this device is offline.',
+	'interactiveProgress.code.runningBootSequence': () => 'Trying to open Suitest bootstrap application.',
+	'interactiveProgress.code.deviceInUse': () => 'A user is currently connected to this device. Execution will continue after the user disconnects.',
+	'interactiveProgress.code.deviceDisabled': () => 'This device is disabled. For the execution to continue please enable the device.',
+	'interactiveProgress.code.deviceDeleted': () => 'Cannot continue: Device is deleted',
+	'interactiveProgress.code.internalError': () => 'Cannot continue: Internal error occurred.',
+	'interactiveProgress.code.notDefinedPlatform': () => 'Cannot continue: Device does not support this platform.',
+	'interactiveProgress.code.lgWebosPlatformError': () => 'Cannot continue: LG WebOS driver failed.',
+	'interactiveProgress.code.xboxPlatformError': () => 'Cannot continue: LG WebOS driver failed.',
```

Comment:
same line as above, is it ok?

Reply:
texts were copied from ticket, fixed without approval